data/allLanguages.json

```diff
@@ -22,5 +22,9 @@
   {
     "code": "id",
     "text": "Indonesia"
+  },
+  {
+    "code": "ru",
+    "text": "русский"
```

Comment:
```suggestion
    "text": "Русский"
```

Reply:
Thank you @lex111 done!

Just to be sure, is the bio at the end OK?

![image](https://user-images.githubusercontent.com/5546996/104588381-9db1be80-56ab-11eb-8e9c-7d7d614709c1.png)

Reply:
Yep, let's merge this.